.buildkite/pipeline.yml

```diff
@@ -0,0 +1,5 @@
+steps:
+  - command: nix-build crossref-verifier.nix
+    label: build everything
+  - command: nix-build
+    label: build default.nix (as if imported externally)
```

Comment:
Does it mean that the same code will be compiled twice or is it somehow shared? I see that `build everything` stage compiled Haskell code and took 18 sec, then `build default.nix` took 12 sec and I don't know what it did (there's almost nothing in its logs). So does `build default.nix` rebuild anything or did it spend 12 sec on some meta stuff?

P. S. I assume the latter, but I think it's fine anyway as long as the whole build takes less than a minute.

Reply:
The code is obviously shared. The main point is to guarantee that importing `default.nix` works. This can perhaps be improved a bit, wait a second.

Reply:
Now it should make more sense.